demogen/models/nin.py

```diff
@@ -85,7 +85,7 @@ def _dropout_block(input_tensor, dropout_prob, spatial):
   """
   shape = input_tensor.shape
   ns = [shape[0], 1, 1, shape[3]] if spatial else None
-  out = tf.nn.dropout(input_tensor, dropout_prob, noise_shape=ns)
+  out = tf.nn.dropout(input_tensor, rate=dropout_prob, noise_shape=ns)
```

Comment:
According to the docs for [`tf.nn.dropout`](https://www.tensorflow.org/api_docs/python/tf/nn/dropout):

```
tf.nn.dropout(
    x, rate, noise_shape=None, seed=None, name=None
)
```

`rate` is a positional variable and does not need a keyword parameter.`dropout_prob` is already being parsed correctly as `rate` without the need for `rate=dropout_prob`

Reply:
Thanks for your reply!

I agree, in Tensorflow 2.x `dropout_prob` would be parsed as `rate`.

The [requirements of the DEMOGEN project ](https://github.com/google-research/google-research/blob/master/demogen/requirements.txt#L1)seem to ask for Tensorflow 1.x, however.

In TF 1.x, the signature of [tf.nn.dropout](https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/nn/dropout) was: 
```
tf.nn.dropout(
    x, keep_prob=None, noise_shape=None, seed=None, name=None, rate=None
)
```

and `dropout_prob` will be parsed as `keep_prob` if it's passed without a keyword.